flambe/optim/noam.py

```diff
@@ -53,4 +53,4 @@ def lr_lambda(self, step: int) -> float:
             if step > self.warmup:
                 return 1. / (self.d_model ** 0.5) / (step ** 0.5)
             else:
-                return step / (self.d_model ** 0.5) / (self.warmup ** 1.5)
+                return (step + 1) / (self.d_model ** 0.5) / (self.warmup ** 1.5)
```

Comment:
This is not part of the canonical implementation by the original authors. I don't think we should change that for consistency.